src/LibraryManager.Vsix/UI/Extensions/RemoveSubstringExtension.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Windows.Markup;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Extensions
+{
+    [MarkupExtensionReturnType(typeof(string))]
+    internal class RemoveSubstringExtension : MarkupExtension
```

Comment:
I'm wondering if we should rename this and also tweak the design.  Right now it removes a substring (by default, a single character, but it can take an arbitrary string).  Based on our offline conversation, it seems like it would more useful in the case we need if it would remove any character in that string.  E.g. if I wanted to remove the angle brackets in `<foo>`, the current implementation would require that I call this separately for `<` and `>`.

What do you think about renaming to `RemoveCharacterExtension` instead, and change the `string.Replace` call on line 46 with a loop over each of the characters in `_remove`?  `RemoveSubstring` could be a useful operation, but it's not serving our current need.

Reply:
I'll be sending out another pr with the suggested design changes.